src/webviews/Document/DocumentPanel.tsx

```diff
@@ -6,21 +6,28 @@
 import { type JSONObject, type PartitionKeyDefinition } from '@azure/cosmos';
 import { makeStyles, MessageBar, ProgressBar } from '@fluentui/react-components';
 import { parse as parseJson } from '@prantlf/jsonlint';
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
+import { useHotkeys } from 'react-hotkeys-hook';
 import { extractPartitionKey } from '../../utils/document';
 import { MonacoEditor } from '../MonacoEditor';
 import { DocumentToolbar } from './DocumentToolbar';
 import { useDocumentDispatcher, useDocumentState } from './state/DocumentContext';
+import { UnsavedChangesDialog } from './UnsavedChangesDialog';
 
 const useClasses = makeStyles({
     resultDisplay: {
-        marginTop: '10px',
-        width: '100%',
-        height: 'calc(100% - 50px)', // Toolbar height is 40px + 10px margin
+        flexShrink: 1,
+        flexGrow: 1,
+        position: 'relative',
     },
     container: {
         marginTop: '10px',
         height: 'calc(100% - 10px)', // 10px margin
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'stretch',
+        alignItems: 'stretch',
+        gap: '5px',
     },
```

Comment:
did it help with monaco's height updates as well?

Reply:
Probably yes. I've tested different ways to change size and it looks as work approach